test/steps/kogitoapp.go

```diff
@@ -30,13 +29,13 @@ var assist = assistdog.NewDefault()
 // RegisterCliSteps register all CLI steps existing
 func registerKogitoAppSteps(s *godog.Suite, data *Data) {
 	// Deploy steps
-	s.Step(`^"([^"]*)" deploy quarkus example service "([^"]*)" with native "([^"]*)"$`, data.deployQuarkusExampleServiceWithNative)
-	s.Step(`^"([^"]*)" deploy quarkus example service "([^"]*)" with native "([^"]*)" and labels$`, data.deployQuarkusExampleServiceWithNativeAndLabels)
-	s.Step(`^"([^"]*)" deploy quarkus example service "([^"]*)" with native "([^"]*)" and persistence$`, data.deployQuarkusExampleServiceWithNativeAndPersistence)
-	s.Step(`^"([^"]*)" deploy quarkus example service "([^"]*)" with native "([^"]*)" and persistence and events$`, data.deployQuarkusExampleServiceWithNativeAndPersistenceAndEvents)
-	s.Step(`^"([^"]*)" deploy spring boot example service "([^"]*)"$`, data.deploySpringBootExampleService)
-	s.Step(`^"([^"]*)" deploy spring boot example service "([^"]*)" with persistence$`, data.deploySpringBootExampleServiceWithPersistence)
-	s.Step(`^"CR" deploy service from example file "([^"]*)"$`, data.deployServiceFromExampleFile)
+	s.Step(`^((?:CR|CLI)?)[\s]*Deploy quarkus example service "([^"]*)" with native ((?:enabled|disabled){1})$`, data.deployQuarkusExampleServiceWithNative)
```

Comment:
What is the purpose of {1} and why do you use ?: to not capture enabled|disabled ?

Reply:
overthinking on my side ? ...

Reply:
alos not sure now that `^((?:CR|CLI)?)[\s]*` is needed anymore ...

Reply:
Yes, the ((?:CR|CLI)?)[\s]* was my second question once we finish previous one :) 
I think that ((?:CR|CLI)?) is needed to express that CR|CLI is not mandatory and the result is captured in every case. AFAIK [\s]* could be dropped.

Reply:
as we don't mention CR or CLI anymore into feature files, I don't think we need to keep the regexp. The choice CR/CLI is done into the code itself.
Also `CR Deploy service from example file` could be directly `Deploy service from example file` as anyway it will be via CR.

Reply:
Ok for me.